src/pages/SudokuPage.js

```diff
@@ -43,20 +43,20 @@ export default function SudokuPage() {
 
   return (
     <>
-      {isLoading || !numbers ? (
+      {isLoading || !sudoku ? (
         <p>Loading...</p>
       ) : (
         <SudokuGrid
-          initialSudokuNumbers={initialNumbers.current}
-          currentSudokuNumbers={numbers}
+          initialSudokuNumbers={initialSudoku.current}
+          currentSudokuNumbers={sudoku}
           onCellClick={handleCellClick}
         />
       )}
       <NumberInputField onNumberInputClick={handleNumberInputClick} />
       <ValidateBtn
         value={"Submit"}
         onClick={handleSubmitClick}
-        numbersArray={numbers}
+        validateData={sudoku}
         url={"https://sugoku.herokuapp.com/validate"}
       />
       <p>{sudokuStatus}</p>
```

Comment:
that could have an h-tag... like h5 or h6... since its not just a "plain" paragraph text.